docs/getting-started-guides/scratch.md

```diff
@@ -514,38 +515,18 @@ To run an etcd instance:
 1. make any modifications needed
 1. start the pod by putting it into the kubelet manifest directory
 
-### Apiserver
+### Apiserver, Controller Manager, and Scheduler
```

Comment:
The numbered list immediately before this section heading renders as all 1's. (To run an etcd instance: 1. copy cluster/saltbase/salt/etcd/etcd.manifest 1. make any modifications needed 1. start the pod by putting it into the kubelet manifest directory) I'm not sure why, maybe you need a newline between "To run an etcd instance" and the first "1."?